capstone-rs/src/instruction.rs

```diff
@@ -157,12 +159,54 @@ impl<'a> Drop for Instructions<'a> {
     }
 }
 
+pub trait Instruction {
```

Comment:
```suggestion
pub unsafe trait Instruction {
```

or seal it; since an evil implementation could provide a silly `.insn()` implementation